BridgeAppSDK/SBAUserWrapper+Bridge.swift

```diff
@@ -517,7 +517,14 @@ public extension SBAUserWrapper {
         }
     }
     
-    fileprivate func updateFromUserSession(_ response: SBAUserSessionInfoWrapper) {
+    /**
+     Call this any time the user session info gets updated.
+     */
+    public func updateFromUserSessionInfo(_ info: SBBUserSessionInfo) {
+        self.updateFromUserSession(info)
+    }
+    
+    private func updateFromUserSession(_ response: SBAUserSessionInfoWrapper) {
```

Comment:
Did this because SBAUserSessionInfoWrapper is an internal protocol. Would it make more sense to make that protocol public?